public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
```

Comment:
You'd have to create the html server side, which would be a pain. Just checking if it's acceptable to not have that feature available when JS is disabled. It's more of a nice to have I'd have thought.